### PR TITLE
increase scalatest patience

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by raboof** [comment]
the travis push job failed.. restarted it.

File: core/src/test/scala/akka/persistence/cassandra/query/EventsByPersistenceIdSpec.scala [hunk 1]
```diff
@@ -53,6 +54,8 @@ class EventsByPersistenceIdSpec
 
   implicit val mat = ActorMaterializer()(system)
 
+  implicit val patience = PatienceConfig(Span(10, Seconds), Span(1, Second))
```
Comment (raboof) [inline core/src/test/scala/akka/persistence/cassandra/query/EventsByPersistenceIdSpec.scala]: Perhaps use named parameters, unclear what's being set to 10 and 1 second now.

**Comment by patriknw** [comment]
something is really flaky with the travis builds, e.g. NoHostAvailableException: All host(s) tried for query failed (tried: /127.0.0.1:35435

**Comment by patriknw** [comment]
seems like this change made the tests more flaky, other PRs are passing, I don't understand

**Pull request merged**